README.md

```diff
@@ -71,6 +69,23 @@ distribution:
             ttl: 10
 ```
 
+#### Lambda@Edge
+
+```yml
+# serverless.yml
+
+distribution:
+  component: '@serverless/aws-cloudfront'
+  inputs:
+    origins:
+      - url: https://sampleorigin.com
+        pathPatterns:
+          /sample/path:
+            ttl: 10
+            lambda@edge:
+              viewer-request: arn:aws:lambda:us-east-1:123:function:myFunc:version # lambda ARN including version
```

Comment:
is the version a strict requirement for Lambda@Edge? I don't think we support versioning in our lambda component 🤔 ... In that case, this component wouldn't be composable with a lambda/function component cause they output a simple versionless ARN.

This effectively means that users of this component could only use it with an already deployed lambda via another tool, which kinda defeats the purpose of the whole thing. What you think? 🤔

Reply:
Unfortunately it must be the ARN of a function version and $LATEST can't be used either. How difficult would it be to add versioning onto the `aws-lambda` component?